CRITICAL - 2021-12-23 10:53:30 --> Call to a member function set() on null
#0 C:\xampp\htdocs\CI\system\View\View.php(214): include()
#1 C:\xampp\htdocs\CI\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#2 C:\xampp\htdocs\CI\system\Common.php(1097): CodeIgniter\View\View->render('items_cart/admi...', Array, true)
#3 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(77): view('items_cart/admi...', Array)
#4 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#5 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#6 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-12-23 11:46:06 --> unlink(): http does not allow unlinking
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(): http ...', 'C:\\xampp\\htdocs...', 51, Array)
#1 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(51): unlink('http://localhos...')
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#4 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-12-23 11:47:32 --> unlink(): http does not allow unlinking
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(): http ...', 'C:\\xampp\\htdocs...', 51, Array)
#1 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(51): unlink('http://localhos...')
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#4 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-12-23 11:50:28 --> unlink(): http does not allow unlinking
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(): http ...', 'C:\\xampp\\htdocs...', 52, Array)
#1 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(52): unlink('http://localhos...')
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#4 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-12-23 11:51:09 --> unlink(C:
mpp\htdocs\CI\public\upload\'upload/1640083762_6dbb477fa527c5c3890c.jpg'): No such file or directory
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(C:\nmpp\\h...', 'C:\\xampp\\htdocs...', 52, Array)
#1 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(52): unlink('C:\nmpp\\htdocs\\C...')
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#4 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-12-23 11:52:03 --> unlink(C:/xampp/htdocs/CI/public/'upload/1640083762_6dbb477fa527c5c3890c.jpg'): No such file or directory
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(C:/xampp...', 'C:\\xampp\\htdocs...', 52, Array)
#1 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(52): unlink('C:/xampp/htdocs...')
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#4 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-12-23 11:55:08 --> unlink(C:/xampp/htdocs/CI/public/upload/1640083762_6dbb477fa527c5c3890c.jpg): No such file or directory
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(C:/xampp...', 'C:\\xampp\\htdocs...', 52, Array)
#1 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(52): unlink('C:/xampp/htdocs...')
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#4 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-12-23 12:16:19 --> syntax error, unexpected 'oldfile' (T_STRING), expecting ';' or ','
#0 C:\xampp\htdocs\CI\system\Router\Router.php(176): CodeIgniter\Router\Router->autoRoute('practicecontrol...')
#1 C:\xampp\htdocs\CI\system\CodeIgniter.php(718): CodeIgniter\Router\Router->handle('practicecontrol...')
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(359): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-12-23 12:29:37 --> unlink(C:/xampp/htdocs/CI/public/upload/1640083762_6dbb477fa527c5c3890c.jpg): No such file or directory
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(C:/xampp...', 'C:\\xampp\\htdocs...', 52, Array)
#1 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(52): unlink('C:/xampp/htdocs...')
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#4 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-12-23 12:35:34 --> unlink(): http does not allow unlinking
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(): http ...', 'C:\\xampp\\htdocs...', 68, Array)
#1 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(68): unlink('http://localhos...')
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#4 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-12-23 12:36:33 --> Cannot access protected property CodeIgniter\Database\MySQLi\Builder::$db
#0 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(68): App\Models\PracticeModel->updateData(Array, 'item_table', '14')
#1 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-12-23 12:41:26 --> Cannot access protected property CodeIgniter\Database\MySQLi\Builder::$db
#0 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(68): App\Models\PracticeModel->updateData(Array, 'item_table', '14')
#1 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-12-23 12:42:29 --> Cannot access protected property CodeIgniter\Database\MySQLi\Builder::$db
#0 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(68): App\Models\PracticeModel->updateData(Array, 'item_table', '14')
#1 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-12-23 12:45:37 --> The original file is not a valid file.
#0 C:\xampp\htdocs\CI\system\HTTP\Files\UploadedFile.php(137): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidFile()
#1 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(59): CodeIgniter\HTTP\Files\UploadedFile->move('C:\\xampp\\htdocs...', '1640285137_bb6d...')
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#4 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-12-23 12:50:48 --> Cannot access protected property CodeIgniter\Database\MySQLi\Builder::$db
#0 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(69): App\Models\PracticeModel->updateData(Array, 'item_table', '14')
#1 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-12-23 12:54:37 --> The original file is not a valid file.
#0 C:\xampp\htdocs\CI\system\HTTP\Files\UploadedFile.php(137): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidFile()
#1 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(59): CodeIgniter\HTTP\Files\UploadedFile->move('C:\\xampp\\htdocs...', '1640285677_ae1c...')
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#4 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-12-23 12:54:51 --> Cannot access protected property CodeIgniter\Database\MySQLi\Builder::$db
#0 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(69): App\Models\PracticeModel->updateData(Array, 'item_table', '14')
#1 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-12-23 13:00:08 --> The original file is not a valid file.
#0 C:\xampp\htdocs\CI\system\HTTP\Files\UploadedFile.php(137): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidFile()
#1 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(59): CodeIgniter\HTTP\Files\UploadedFile->move('C:\\xampp\\htdocs...', '1640286008_c55d...')
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#4 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-12-23 13:00:39 --> Cannot access protected property CodeIgniter\Database\MySQLi\Builder::$db
#0 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(69): App\Models\PracticeModel->updateData(Array, 'item_table', '14')
#1 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-12-23 13:11:49 --> Illegal string offset 'category'
#0 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(74): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Illegal string ...', 'C:\\xampp\\htdocs...', 74, Array)
#1 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-12-23 13:12:52 --> Illegal string offset 'category'
#0 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(74): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Illegal string ...', 'C:\\xampp\\htdocs...', 74, Array)
#1 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-12-23 13:13:31 --> Illegal string offset 'category'
#0 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(74): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Illegal string ...', 'C:\\xampp\\htdocs...', 74, Array)
#1 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-12-23 13:14:44 --> Illegal string offset 'category'
#0 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(74): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Illegal string ...', 'C:\\xampp\\htdocs...', 74, Array)
#1 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-12-23 17:15:35 --> session_destroy(): Trying to destroy uninitialized session
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'session_destroy...', 'C:\\xampp\\htdocs...', 25, Array)
#1 C:\xampp\htdocs\CI\app\Controllers\PracticeController.php(25): session_destroy()
#2 C:\xampp\htdocs\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictAdmin()
#3 C:\xampp\htdocs\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#4 C:\xampp\htdocs\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
ERROR - 2021-12-23 10:18:20 --> Unknown column 'itm_name' in 'field list'
CRITICAL - 2021-12-23 10:18:20 --> Call to a member function getResult() on bool
#0 C:\wamp64\www\CI\app\Controllers\PracticeController.php(20): App\Models\PracticeModel->getDataField('cate_table', 'itm_name,itm_pr...')
#1 C:\wamp64\www\CI\system\CodeIgniter.php(824): App\Controllers\PracticeController->redictUsr()
#2 C:\wamp64\www\CI\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PracticeController))
#3 C:\wamp64\www\CI\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\wamp64\www\CI\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
